images/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Images"
 icon: "<svg viewBox=\"0 0 16 16\" xmlns=\"http://www.w3.org/2000/svg\">\n<path d=\"M15.0017 4.0017H1.0017C0.868289 3.99368 0.734692 4.01405 0.609732 4.06147C0.484772 4.10889 0.371292 4.18228 0.276784 4.27678C0.182276 4.37129 0.108891 4.48477 0.0614728 4.60973C0.0140548 4.73469 -0.00631686 4.86829 0.00170278 5.0017V15.0017C-0.00631686 15.1351 0.0140548 15.2687 0.0614728 15.3937C0.108891 15.5186 0.182276 15.6321 0.276784 15.7266C0.371292 15.8211 0.484772 15.8945 0.609732 15.9419C0.734692 15.9894 0.868289 16.0097 1.0017 16.0017H15.0017C15.1351 16.0097 15.2687 15.9894 15.3937 15.9419C15.5186 15.8945 15.6321 15.8211 15.7266 15.7266C15.8211 15.6321 15.8945 15.5186 15.9419 15.3937C15.9894 15.2687 16.0097 15.1351 16.0017 15.0017V5.0017C16.0097 4.86829 15.9894 4.73469 15.9419 4.60973C15.8945 4.48477 15.8211 4.37129 15.7266 4.27678C15.6321 4.18228 15.5186 4.10889 15.3937 4.06147C15.2687 4.01405 15.1351 3.99368 15.0017 4.0017ZM14.0017 14.0017H2.0017V6.0017H14.0017V14.0017Z\" />\n<path d=\"M14 0H2V2H14V0Z\" fill=\"#333333\"/>\n<path d=\"M5 13L10 8L13 13H5Z\" />\n<path d=\"M5 10C5.55228 10 6 9.55228 6 9C6 8.44772 5.55228 8 5 8C4.44772 8 4 8.44772 4 9C4 9.55228 4.44772 10 5 10Z\" />\n</svg>"
-description: Learn about the base of reproducibility inside Coder.
+description: Learn about the basis of reproducibility inside Coder.
 ---
 
 Images are the basis from which new [Environments](../environments/index.md)
-are created. They are the base of Environment reproducibility.
+are created. They are the basis of environment reproducibility.
```

Comment:
Coder creates [Environments] using templates called _images_, which encourage reproducibility of development environments.

If we haven't already, we should blog about the benefits of reproducible software development practices, as I think that's a key differentiator for us as compared to VDI solutions like Citrix